### PR TITLE
Crowbar POC initial version

### PR DESCRIPTION
**Pull request by dmpop**

### CODE REVIEW

File: xml/book_poc_crowbar.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:lang="en"
+      xml:id="book.poc.crowbar">
+ <title>&crow; Proof of Concept Guide</title>
+ <info>
+ <xi:include href="copyright_suse_cloud.xml"/>
+  <abstract>
+   <para>
+     At the time of the &productnumber; release, this guide
+     is a proof of concept (POC) of a &productname; deployment.
```
Comment (csymons-suse) [inline xml/book_poc_crowbar.xml]: is a proof of concept (POC) of a &productname; deployment.

should be an &productname;

File: xml/book_poc_crowbar.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+     This guide provides the basic steps for deployment
+     &productname; &productnumber; in a bare metal environment.
```
Comment (csymons-suse) [inline xml/book_poc_crowbar.xml]: for deployment
maybe deploying
or
deployment of

Comment (dmpop) [reply xml/book_poc_crowbar.xml]: I opted for _This guide describes a basic deployment of_ How does this sound?

Comment (csymons-suse) [reply xml/book_poc_crowbar.xml]: sounds fine.

**Pull request merged**